python/geospark/core/jvm/config.py

```diff
@@ -65,8 +65,9 @@ def get_used_jars():
         java_spark_conf = spark_conf._jconf
         try:
             used_jar_files = java_spark_conf.get("spark.jars")
-        except Exception as e:
-            used_jar_files = ",".join(os.listdir(os.path.join(os.environ["SPARK_HOME"], "jars")))
+        finally:
+            if not used_jar_files:
+                used_jar_files = ",".join(os.listdir(os.path.join(os.environ["SPARK_HOME"], "jars")))
```

Comment:
I also think that's a good practice. Will revise the PR soon.

Reply:
Updated the PR. I think that java.util.NoSuchElementException can be raised from Py4J here, so I used Exception rather than KeyException to catch all types of exceptions.